.eslintignore

```diff
@@ -2,6 +2,6 @@
 /flow-typed/**
 lib/
 es/
-examples/
+examples/*/token.json
```

Comment:
may as well lint the examples here?

Reply:
ya, i jsut didn't want to spend time on it when i set it up initially